src/istio/common/manifests/pepr-istio-config.yaml

```diff
@@ -21,3 +21,18 @@ spec:
   portLevelMtls:
     "3000":
       mode: PERMISSIVE
+---
+apiVersion: "security.istio.io/v1beta1"
+kind: PeerAuthentication
+metadata:
+  name: permissive-pepr-webhook-watcher
+  namespace: pepr-system
+spec:
+  selector:
+    matchLabels:
+      pepr.dev/controller: watcher
+  mtls:
+    mode: STRICT
+  portLevelMtls:
+    "3000":
+      mode: PERMISSIVE
```

Comment:
Just for my memory, is this primarily to handle metrics?

Reply:
Yes